app/src/main/java/com/isc/hermes/controller/TrafficAutomaticFormController.java

```diff
@@ -104,7 +104,7 @@ public int uploadTrafficDataBase(){
         String dateCreated = IncidentsUtils.getInstance().generateCurrentDateCreated();
         String deathDate = IncidentsUtils.getInstance().addTimeToCurrentDate(getTrafficTime());
         String coordinates = TrafficUploader.getInstance().getCoordinates();
-        String JsonString = TrafficUploader.getInstance().generateJsonIncident(id,getTrafficType(20,50),"Traffic Test",dateCreated, deathDate , GeometryType.LINE_STRING.getName(),coordinates);
+        String JsonString = TrafficUploader.getInstance().generateJsonIncident(getTrafficType(20,50),"Traffic Test",dateCreated, deathDate , GeometryType.LINE_STRING.getName(),coordinates);
```

Comment:
The "reason" parameter received by this method should be set only as traffic and not traffic test.

Reply:
done